cmd/podman/wait.go

```diff
@@ -15,7 +16,14 @@ var (
 
 	Block until one or more containers stop and then print their exit codes
 `
-	waitFlags   = []cli.Flag{LatestFlag}
+	waitFlags = []cli.Flag{
+		cli.UintFlag{
+			Name:  "interval, i",
+			Usage: "Microseconds to wait before polling for completion",
```

Comment:
whats the default?

Reply:
I think we're better off setting this to milliseconds - 1000 times a second should be plenty

Reply:
Yup